internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/PartialUpdateTest.java

```diff
@@ -140,6 +141,97 @@ public void setUp() {
     this.parentController = parentControllers.get(0);
   }
 
+  @Test
+  public void testTest() {
```

Comment:
Can we have a better name on this?

Reply:
Oh right this test should not be part of the PR, I will remove it.